### PR TITLE
Improve batcher workflow metrics

### PR DESCRIPTION
**Pull request by yux0**
**What changed?**
1. Improve batcher workflow metrics.
2. Add frontend batch operation API feature flag.
3. Enable batcher per ns worker by default.

**Why?**
Improve batcher workflow metrics

**How did you test it?**
Local test

**Potential risks**

**Is hotfix candidate?**

### CODE REVIEW

File: service/frontend/service.go
Comment (dnr) [inline service/frontend/service.go:150]:
```suggestion
	// Enable batcher RPCs
```

File: common/dynamicconfig/constants.go
Comment (dnr) [inline common/dynamicconfig/constants.go:95]: maybe
```suggestion
	EnableBatcher = "system.enableBatcher"
```
?

Comment (dnr) [reply common/dynamicconfig/constants.go]: It's still possible to enable/disable each worker component (scheduler, batcher) per-namespace, even though you can't change the count. So we could have another dynamic config for that. I don't feel strongly either way.

File: service/worker/batcher/activities.go
Comment (dnr) [inline service/worker/batcher/activities.go:71]: why not just use `batchParams.Namespace`?

**Review by dnr — APPROVED** [review]
also add a line to the PR description about the frontend dynamic config

File: Makefile
Comment (alexshtin) [inline Makefile:120]: I released new version of plugin. Please revert this change and rebase after #3300 is merged.

File: service/worker/batcher/activities.go
Comment (yux0) [reply service/worker/batcher/activities.go:71]: Updated

File: Makefile
Comment (yux0) [reply Makefile]: Sure. I reverted the commit and wait for the PR to merge.

**Pull request merged**
